src/main/java/File/FileManager.java

```diff
@@ -0,0 +1,222 @@
+            sc.next(); // skip the symbols
+            String current = sc.next();
+            while (current.charAt(0) != '(') {
+                name = name + current + " ";
+                current = sc.next();
+            }
+        } catch (NoSuchElementException e) {
+            // Do nothing
+        }
+        return name;
+    }
+
+    /**
+     * Returns the date and time of the line in the file.
+     *
+     * @param s text.
+     * @return String date and time in String.
+     * @throws ErrorExceptions error reading the date and time.
+     */
+    private static String getDate(String s) throws ErrorExceptions {
+        Scanner sc = new Scanner(s);
+        String date = "";
+        try {
+            String current = sc.next();
+            while (current.charAt(0) != '(') {
+                current = sc.next();
+            }
+            current = sc.next();
+            while (current.charAt(current.length() - 1) != ')') {
+                date = date + current + " ";
+                current = sc.next();
+            }
+            int l = current.length() - 1;
+            String last = "";
+            for (int i = 0; i < l; i++) {
+                last = last + current.charAt(i);
+            }
+            date = date + last;
+        } catch (NoSuchElementException e) {
+            System.out.println(e);
+        }
+        DateTimeFormatter d = DateTimeFormatter.ofPattern("dd MMM yyyy HHmm");
+        LocalDateTime dt = LocalDateTime.parse(date, d);
+        return dt.format(DateTimeFormatter.ofPattern("dd-MM-uuuu HHmm"));
+    }
+
+    /**
+     * Returns the type of task in the file.
+     *
+     * @param s text line.
+     * @return int type of task.
+     * @throws ErrorExceptions error reading the type in the file.
+     */
+    private static int getType(String s) throws ErrorExceptions {
+        Scanner sc = new Scanner(s);
+        try {
+            String current = sc.next();
+            char type = current.charAt(1);
+            if (type == 'T') {
+                return 1;
+            } else if (type == 'D') {
+                return 2;
+            } else if (type == 'E') {
+                return 3;
+            } else {
+                throw new ErrorExceptions("Error: Wrong item type detected, file might be corrupted!");
+            }
+        } catch (NoSuchElementException e) {
+            throw new ErrorExceptions("Failed to load saved file (Type)" + System.lineSeparator()
+                    + e);
+        }
+    }
+
+    /**
+     * Returns whether the task is completed.
+     *
+     * @param s text line.
+     * @return boolean completed or not completed.
+     * @throws ErrorExceptions error when reading the file.
+     */
+    private static boolean getDone(String s) throws ErrorExceptions {
+        Scanner sc = new Scanner(s);
+        try {
+            String current = sc.next();
+            char done = current.charAt(4);
+            String d = "";
+            d = d + done;
```

Comment:
I've noticed this in other places as well but perhaps some variable names could be longer to give an indication of their purpose? In this case maybe the `done` variable should have been the `String` which might help in reading through the code.

E.g.
```
char d = current.charAt(4);
String done = "";
done = done + d;
```

This might help in the latter parts when referring to what the variable means.